charts/leonardo/Chart.yaml

```diff
@@ -11,3 +11,8 @@ sources:
   - https://github.com/broadinstitute/terra-helm/
   - https://github.com/DataBiosphere/leonardo
   - https://github.com/broadinstitute/leonardo-cron-jobs
+dependencies:
+  - name: liquibase-migration
+    condition: migration.enabled
```

Comment:
I don't know helm well, but is it possible to have this condition be something like `liquibaseCommand == updateSQL`?

Reply:
Since `enabled` controls whether liquibase k8s resources are created, I think that's the correct flag to import the library chart based on. `liquibaseCommand` is going to be either `update` or `updateSQL` for whether we want real operation or dry-run. Just updated the comment in the values file for that too, might be a bit clearer.